Assets/Sanji.cs

```diff
@@ -13,6 +14,8 @@ public class Sanji : Character {
 
 		player.knockbackVector = new Vector2 (direction > 0 ? -1 : 1, 1);
 		player.wasThrown (direction);
+
+		hitter.GetComponent<Health> ().Damage (kickDmg);
```

Comment:
But the PlayerMovement script should encapsulate movement and not have the dependency of health. I think it's better to leave the Damage() call in Sanji.cs